content/docs/workflows/m2k.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: serverless-workflows m2k
+title: Move To Kube (m2k)
```

Comment:
I think the official name is move2kube (https://move2kube.konveyor.io/)

Reply:
I'm looking for some title to it. I'll make it move2kube but if you have better titles I think we should use them